packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/index.hbs

```diff
@@ -1,19 +1,75 @@
 <ActionCardContainer
+  class="face-value-card"
   @header="Prepaid card funding"
   @isComplete={{@isComplete}}
-  data-test-layout-customization-is-complete={{@isComplete}}
 >
+  <ActionCardContainer::Section @title="Choose the face value of your prepaid card" class="face-value-card__section">
+    <div class="face-value-card__container">
+      <p class="face-value-card__exchange">
+        <span>Prepaid cards are denominated in SPEND:</span>
+        <span class="face-value-card__exchange-rate">
+          {{svg-jar "spend" width="20" height="20" role="presentation"}}
+          §1 SPEND = ${{this.spendToUsdRate}} USD
+        </span>
+      </p>
+      <CardPay::LabeledValue @vertical={{true}} @label="Funding From:">
+        <CardPay::IssuePrepaidCardWorkflow::FaceValue::BalanceView
+          @walletAddress={{this.layer2Network.walletInfo.firstAddress}}
+          @depotAddress={{this.layer2Network.depotSafe.address}}
+          @token={{this.fundingToken}}
+          @balance={{format-token-amount this.fundingToken.balance}}
+        />
+      </CardPay::LabeledValue>
+      {{#if @isComplete}}
+        <CardPay::LabeledValue @vertical={{true}} @label="Face Value:">
+          <CardPay::BalanceDisplay
+            class="face-value-card__selected-amount-display"
+            @size="large"
+            @icon="spend"
+            @sign="§"
+            @symbol="SPEND"
+            @balance={{this.selectedFaceValue}}
+            @usdBalance={{spend-to-usd this.selectedFaceValue}}
+            @text={{concat "≈ " (spend-to-usd this.selectedFaceValue) " " this.fundingToken.symbol "*"}}
+            data-test-face-value-display
+          />
+        </CardPay::LabeledValue>
+      {{else}}
+        <div class="face-value-card__options">
+          {{#each this.faceValueOptions as |amount|}}
+            <CardPay::IssuePrepaidCardWorkflow::FaceValue::Option
+              @disabled={{lt this.balanceFromBN (spend-to-usd amount)}}
+              @checked={{eq amount this.selectedFaceValue}}
+              @spendFaceValue={{amount}}
+              @usdAmount={{spend-to-usd amount}}
+              @approxDaiAmount={{spend-to-usd amount}}
```

Comment:
Approximate DAI amount can probably use the ExchangeRate.convertFromSpend method from the SDK?

Reply:
Yes, I got that to work in a future commit but requires some clean up